### PR TITLE
SBEE-33: Update the Getting Started doc page for CSB 1.1

### PR DESCRIPTION
**Pull request by eric-schneider**
[SBEE-33](https://issues.couchbase.com/browse/SBEE-33): Update the Getting Started doc page for CSB 1.1

### CODE REVIEW

File: modules/ROOT/pages/install/get-started.adoc [hunk 1]
```diff
@@ -1,29 +1,17 @@
-= Install and Get Started with Couchbase Service Broker on Kubernetes
+= Get Started with Couchbase Service Broker on Kubernetes
+include::partial$constants.adoc[]
 :page-aliases: install:container.adoc, install:index.adoc, install:ingress.adoc, install:kubernetes.adoc, install:packages.adoc, install:prerequisites.adoc, install:serviceinstance.adoc
 
 [abstract]
-This section describes how to get and install the service broker and create your first service instance.
+This page describes how to get and install the Couchbase Service B roker and create your first service instance.
```
Comment (spjmurray) [inline modules/ROOT/pages/install/get-started.adoc]: typo

File: modules/ROOT/pages/install/get-started.adoc [hunk 2]
```diff
@@ -36,69 +24,124 @@ In general, any Kubernetes distribution based on a supported version should work
...
+The Service Broker can be installed at the namespace level or the cluster level.
+
+[{tabs}]
 ====
+namespace::
++
+--
+[source,console]
+----
+$ ./cbsbctl create clusterservicebroker --profile demo-servicebroker
```
Comment (spjmurray) [inline modules/ROOT/pages/install/get-started.adoc]: profile is just "demo"

File: modules/ROOT/pages/install/get-started.adoc [hunk 3]
```diff
@@ -36,69 +24,124 @@ In general, any Kubernetes distribution based on a supported version should work
...
+----
+--
 
+cluster::
++
+--
 [source,console]
 ----
-$ kubectl create -f broker.yaml
+$ ./cbsbctl create clusterservicebroker --profile demo-clusterservicebroker
```
Comment (spjmurray) [inline modules/ROOT/pages/install/get-started.adoc]: profile is just "demo"

File: modules/ROOT/pages/install/get-started.adoc [hunk 4]
```diff
@@ -171,13 +219,11 @@ NAME   TYPE     DATA   AGE
 csb    	Opaque       5      7m58s
 ----
 
-To access the couchbase cluster UI console: 
+To access the Couchbase Web Console:
 
 [source,console]
 ----
 $ kubectl port-forward couchbase-instance-winhhoku-0000 8091
```
Comment (spjmurray) [inline modules/ROOT/pages/install/get-started.adoc:226]: Where does thins pod name come from?  Thin air it looks like.  You'll need a way to map your service instance to the pod name or a user is going to complain.

File: modules/ROOT/partials/tools-install.adoc [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+[{tabs}]
+====
+macOS::
++
+--
+. Open a Terminal window and go to the directory where the `cbsbctl` binary is located:
++
+[source,console]
+----
+$ cd couchbase-service-broker-kubernetes_x.x.x-macos_x86_64/
```
Comment (spjmurray) [inline modules/ROOT/partials/tools-install.adoc]: There is only one package and will untar to `couchbase-service-broker-kubernetes` for all operating systems

File: modules/ROOT/partials/tools-install.adoc [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+----
+
+. Make the `cbsbctl` binary executable:
++
+[source,console]
+----
+$ chmod +x ./cbsbctl
```
Comment (spjmurray) [inline modules/ROOT/partials/tools-install.adoc]: `cbsbctl.darwin` is the binary name

File: modules/ROOT/partials/tools-install.adoc [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
++
+--
+. Open a command prompt and go to the directory where the `cbsbctl` binary is located:
++
+[source,console]
+----
+$ cd couchbase-service-broker-kubernetes_x.x.x-windows_x86_64\
+----
+
+. Add the `cbsbctl` binary into your PATH.
```
Comment (spjmurray) [inline modules/ROOT/partials/tools-install.adoc]: `cbsbctl.exe` is the binary name

**Review by spjmurray — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/install/get-started.adoc [hunk 8]
```diff
@@ -114,55 +157,60 @@ These can be found in the `couchbase-service-broker` cluster role binding define
...
+You can run the following command to check the status of the Service Broker:
 
 [source,console]
 ----
 $ kubectl get deployments
-
+----
+----
 NAME                          	 	READY   	UP-TO-DATE   AVAILABLE   	AGE
 couchbase-operator            		 1/1     		1            	1        	33m
```
Comment (eric-schneider) [inline modules/ROOT/pages/install/get-started.adoc:169]: Just noticed this, and it reminded me to point out that I removed instructions to "install the operator" and went with just the explicit instruction to install the CRD and DAC. So I should probably remove this line?

File: modules/ROOT/pages/install/get-started.adoc [hunk 9]
```diff
@@ -171,13 +219,11 @@ NAME   TYPE     DATA   AGE
...
 
 [source,console]
 ----
 $ kubectl port-forward couchbase-instance-winhhoku-0000 8091
 ----
 
-Go to http://localhost:8091 and login with username as Administrator & password as password.
-
-// The following example demonstrates how to deploy a Couchbase cluster on your local with Couchbase Service Broker from scratch.
+Go to `http://localhost:8091` and login with the credentials used when provisioning the Couchbase cluster service.
```
Comment (spjmurray) [inline modules/ROOT/pages/install/get-started.adoc:229]: For the record, this is a complete service broker anti-pattern.  If you want UI access you need to provision an ingress in your templates and return the endpoint address as part of the binding.

**Review by spjmurray — APPROVED** [review]

**Pull request merged**